styles/main.css

```diff
@@ -236,34 +236,35 @@ img {
     header nav ul a{
         border:0;
     }
-    header nav ul a :hover {
+    header nav ul a:hover {

```

Comment:
Don't put spaces between pseudo classes and the selectors you want them to apply to